app/models/test_track/remote/assignment_event.rb

```diff
@@ -3,7 +3,7 @@ class TestTrack::Remote::AssignmentEvent
 
   collection_path '/api/v1/assignment_event'
 
-  attributes :visitor_id, :split_name, :unsynced
+  attributes :visitor_id, :split_name, :unsynced, :context
```

Comment:
`context` is accepted on the other side of the wire, felt like it was worth including here for clarity